types/changelog-parser/index.d.ts

```diff
@@ -0,0 +1,24 @@
+// Type definitions for changelog-parser 2.7
+// Project: https://github.com/hypermodules/changelog-parser
+// Definitions by:  Adam Zerella <https://github.com/adamzerella>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+interface Options {
+    /**
+     * Path to changelog file.
+     */
+    filePath: string;
+    /**
+     * Removes the markdown markup from the changelog entries by default.
+     * You can change its value to false to keep the markdown.
+     */
+    removeMarkdown: boolean;
+}
+
+/**
+ * Change log parser for node.
+ */
+declare function parseChangelog(options: Partial<Options>|string, callback?: (result: object, error: string|null) => void): Promise<object>;
```

Comment:
last thing, is this the right way around? the callback. it seems to be:

`(err, result) => void` in tests
`(result, err) => void` here

need switching around maybe?

Reply:
Updated test arguments, fixed

Reply:
I'm super sorry to comment on this again but just to double check and be 100% certain, are we certain the error comes second?

https://github.com/hypermodules/changelog-parser/blob/master/README.md#callback

This seems to suggest a callback is indeed what the tests had originally: `(err: string|null, result: object) => void`

Reply:
Hi @43081j, I am the author of `changelog-parser`, you are correct, the callback is error first.

Reply:
yep though it looks like its merged already so i suppose a further PR will be needed to correct it. 

thanks for taking the time to confirm